sbc_compassion/tests/test_tools.py

```diff
@@ -121,7 +121,8 @@ def _read_img(self, path):
 
     def _pattern_recognition(self, path):
         img = self._read_img(path)
-        return tools.patternrecognition.find_template(img, self.templates, 0.3)
+        return tools.patternrecognition.find_template(
+            img, self.templates, 0.3)[:2]
```

Comment:
If I am doing it, I will always ignore the last parameter (it will always be None).
Maybe it will be better if I am just explaining in a comment before returning the values